Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -99,7 +99,7 @@ static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const St
 
 static Result<NonnullRefPtr<DynamicLoader>, DlErrorMessage> map_library(const String& name)
 {
-    if (name.contains("/")) {
+    if (name.contains("/"sv)) {
```

Comment:
This could also be just a character, but meh

Reply:
I checked last night, there is no `String::contains()` for just a char? Maybe it's names something else?

Reply:
:thinking:
```c++
    [[nodiscard]] bool contains(char) const;
```

Reply:
Oh that's a `String`.
this is from `StringView`